src/Features/CSharp/Portable/Debugging/CSharpProximityExpressionsService.Worker.cs

```diff
@@ -119,6 +119,12 @@ private void AddMethodParameters()
                     var parameterList = ((MemberDeclarationSyntax)block.Parent).GetParameterList();
                     AddParameters(parameterList);
                 }
+                else if (block is null
+                    && _parentStatement.Parent is GlobalStatementSyntax { Parent: CompilationUnitSyntax compilationUnit } globalStatement
+                    && compilationUnit.Members.FirstOrDefault() == globalStatement)
+                {
+                    _expressions.Add("args");
```

Comment:
feels like these could be in all top-level statements.  but i get the idea that maybe it's not 'close' unless on the first.

Reply:
➡️ I made top-level statements behave exactly the same as the full form. Only showing method parameters on the first statement is part of the design that precedes my changes.